modules/private-clusters-about-aws.adoc

```diff
@@ -24,6 +24,7 @@ The installation program does use the `baseDomain` that you specify to create a
 == Limitations
 
 The ability to add public functionality to a private cluster is limited.
-
```

Comment:
Removing this blank line causes the subsequent list to be misformatted in the Netlify preview: https://52037--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private.html#private-clusters-limitations-aws_installing-aws-private

Reply:
Ok, yeah the duplication is a bit much and I think the one at the top of the page is where its most likely to be noticed anyways so I'll remove that one under the limitations